subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy

```diff
@@ -106,14 +107,11 @@ Root project 'webinar-parent'
 +--- Project ':webinar-impl' - Webinar implementation
 \\--- Project ':webinar-war' - Webinar web application
 """
-
-        where:
-        scriptDsl << ScriptDslFixture.SCRIPT_DSLS
     }
 
-    @ToBeFixedForConfigurationCache(because = "Kotlin Gradle Plugin") // Kotlin compilation is used for the pre-compiled script plugin
+    @ToBeFixedForConfigurationCache(because = "Kotlin Gradle Plugin", bottomSpecs = "KotlinDslMavenConversionIntegrationTest") // Kotlin compilation is used for the pre-compiled script plugin
```

Comment:
Why not enable this annotation for `GroovyDslMavenConversionIntegrationTest`? Also, I guess there's no way to make this string a constant, right?

Reply:
The test works with configuration caching and Groovy. Only Kotlin fails.

Reply:
I extracted the name in a constant.